website/docs/static/luigi-client/package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
this also seems like a file that shouldn't be here. Most likely autogenerated I presume. Include all of them in a gitignore or please let me know why it should be here